app/javascript/src/BackendApis/components/NewBackendForm.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react'
+import { useState } from 'react'
+
+import { ActionGroup, Button, Form } from '@patternfly/react-core'
+import validate from 'validate.js'
+import {
+  NameInput,
+  SystemNameInput,
+  DescriptionInput,
+  PrivateEndpointInput
+} from 'BackendApis'
+import { CSRFToken } from 'utilities/utils'
+
+type Props = {
+  action: string,
+  onCancel: () => void,
+  isLoading?: boolean,
+  errors?: {
+    private_endpoint: Array<string>
+  }
+}
+
+const VALIDATION_CONSTRAINTS = {
+  name: { length: { minimum: 1 } },
+  privateEndpoint: { url: { schemes: ['http', 'https', 'ws', 'wss'] } }
```

Comment:
I think this explains https://issues.redhat.com/browse/THREESCALE-7480 @josemigallas

Reply:
<img width="917" alt="Screenshot 2021-09-06 at 10 41 17" src="https://user-images.githubusercontent.com/11672286/132195538-95698385-cba0-4d73-b637-1d0bad2c4c1d.png">
I can't reproduce the bug